tsl/src/continuous_aggs/create.c

```diff
@@ -1731,3 +1749,363 @@ tsl_process_continuous_agg_viewstmt(ViewStmt *stmt, const char *query_string, vo
 	cagg_create(stmt, query, &timebucket_exprinfo, with_clause_options);
 	return true;
 }
+
+/*
+ * update the view definition of an existing continuous aggregate
```

Comment:
should add a note to the comment that only the "materialization option" can be changed. The actual query definition itself cannot be modified. 
Could you also add an assert to verify this or a comment saying that the caller should verify this.